.vscode/tasks.json

```diff
@@ -0,0 +1,29 @@
+{
+	"version": "2.0.0",
+	"tasks": [{
+		"label": "Start dev container",
+		"type": "shell",
+		"command": "docker compose --file .devcontainer/docker-compose.yaml up -d",
+		"windows": {
+		  "command": "docker compose --file .devcontainer\\docker-compose.yaml up -d"
+		},
```

Comment:
Nitpick: is this key required? I think most shells on windows will pickup the forward slash fine, but let me know if this is actually necessary :smile:

Reply:
It was like this in the VS Code documentation, so I did the same